Editing FAQ. (#17153)
metabase/metabase: docs/faq/using-metabase/how-do-i-answer-questions-with-joins.md

# How do I answer questions where I need to join tables together?

> If you are using Metabase version 0.32 or earlier, you will need to either write a SQL query that contains joins or have your Metabase administrator set up foreign keys: they can read more about that [here][editing-metadata].

Reviewer: ```suggestion
If you're using Metabase version 0.32 or earlier, you'll need to either write a SQL query that contains joins, or have your Metabase administrator set up foreign keys (they can read more about that [here][editing-metadata]).
```
